src/WrapperScripts.sol

```diff
@@ -10,21 +10,21 @@ contract WrapperActions {
         IWETH(weth).deposit{value: amount}();
     }
 
-    function wrapETHUpTo(address weth, uint256 targetAmount) external payable {
-        uint256 currentBalance = IERC20(weth).balanceOf(address(this));
-        if (currentBalance < targetAmount) {
-            IWETH(weth).deposit{value: targetAmount - currentBalance}();
+    function wrapAllETH(address weth) external payable {
+        uint256 ethBalance = address(this).balance;
+        if (ethBalance > 0) {
+            IWETH(weth).deposit{value: ethBalance}();
         }
     }
 
     function unwrapWETH(address weth, uint256 amount) external {
         IWETH(weth).withdraw(amount);
     }
 
-    function unwrapWETHUpTo(address weth, uint256 targetAmount) external {
-        uint256 currentBalance = address(this).balance;
-        if (currentBalance < targetAmount) {
-            IWETH(weth).withdraw(targetAmount - currentBalance);
+    function unwrapAllWETH(address weth) external payable {
```

Comment:
Is unwrap really unwrapAll?

Reply:
Yeah, should be right? I don't see how `unwrap` should be treated differently then `wrap` if we go the `wrap all` route.

Reply:
Based on the offline discussions we had, I changed the QuarkBuilder to now use `unwrapWETHUpTo` for WETH. Every other wrap/unwrap action is still using "all" instead of "up to".